spec/rspec/core/filter_manager_spec.rb

```diff
@@ -292,15 +292,16 @@ def add_filter(options)
       end
 
       context "with multiple inclusion filters" do
-        it 'includes objects that match any of them' do
+        it 'includes objects that match all of them' do
           examples = [
-            included_1 = example_with(:foo => true),
-            included_2 = example_with(:bar => true),
-                         example_with(:bazz => true)
+            included = example_with(:foo => true, :bar => true),
+                       example_with(:foo => true),
+                       example_with(:bar => true),
+                       example_with(:baz => true)
```

Comment:
Similarly we don't need multiple negatives here

Reply:
Kept one that is part of the filter, and another unrelated. Got rid of redundant `baz`.